lib/postProcessLdesConceptualService.js

```diff
@@ -196,6 +196,7 @@ async function updateConceptualService( versionedServiceGraph, versionedServiceU
           }
         }`);
   }
+  await setStatus("updated", versionedServiceUri);
```

Comment:
this will also set 'updated' to all new concepts. Even though by accident, this won't do anything; I'd like to have it clean.
First check if this is a new concept or not. And if not, look for the instances of the public services using this concept.